.circleci/build.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# To be changed with GCS access keys, region, repo name and url. Creating repo and SA in GCS is a prerequisite
+# cli commands to be used is
+function sanity_check() {
+  
+  if [ -z "${GCS_SECRET_ACCESS_KEY}" ]; then
+    echo "GCS_SECRET_ACCESS_KEY found empty. Exiting ..."
+    exit 1
+  fi
+
+  if [ -z "${CYBERDOJO_HELM_REPO_NAME}" ]; then
+    echo "REPO_NAME found empty. Exiting ..."
+    exit 1
+  fi
+
+  if [ -z "${CYBERDOJO_HELM_REPO_BUCKET_NAME}" ]; then
+    echo "REPO_URL found empty. Exiting ..."
+    exit 1
+  fi
+}
+
+echo "performing sanity checks ..."
+sanity_check
+
+echo "initializing helm ..."
+helm init --client-only
```

Comment:
this is a helm v2 command. The dockerfile uses helm v3.2.1

Reply:
Docker file updated to use helm v2

Reply:
Updated docker file to use helm v2